page/generic.scss

```
@@ -72,6 +72,43 @@
+
+.fcitx-tab {
+  display: flex;
+  justify-content: center;
+}
+
+.fcitx-tab-inner {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  white-space: nowrap;
+  /* Tabs should not be rotated by vertical writing mode. */
```

Stylelint (17.14.0)

[error] Expected empty line before comment (comment-empty-line-before)

(comment-empty-line-before)